#!/usr/bin/env python
PACKAGE = "robot_vision"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("h_lower", int_t, 0, "HSV color space h_low", 0, 0, 255)
gen.add("s_lower", int_t, 0, "HSV color space s_low", 0, 0, 255)
gen.add("v_lower", int_t, 0, "HSV color space v_low", 0, 0, 255)
gen.add("h_upper", int_t, 0, "HSV color space h_high", 255, 0, 255)
gen.add("s_upper", int_t, 0, "HSV color space s_high", 255, 0, 255)
gen.add("v_upper", int_t, 0, "HSV color space v_high", 255, 0, 255)

# gen.add("v0", double_t, 0, "v0", 50.0, 30.0, 120.0)

exit(gen.generate(PACKAGE, "robot_vision", "line_hsv"))
